pkg/controller/predicates/resources/resources.go

```diff
@@ -117,3 +118,27 @@ var DSCComponentUpdatePredicate = predicate.Funcs{
 		return false
 	},
 }
+
+var DSCIReadiness = predicate.Funcs{
+	UpdateFunc: func(e event.UpdateEvent) bool {
+		oldObj, ok := e.ObjectOld.(*dsciv1.DSCInitialization)
+		if !ok {
+			return false
+		}
+		newObj, ok := e.ObjectNew.(*dsciv1.DSCInitialization)
+		if !ok {
+			return false
+		}
```

Comment:
i am fine with this for now.
we might eventually need to revisit the status of DSCI.
currently, it is "in progress" for the very first time, then "error" or "ready" 
so in the case of "ready"->"error" might not need act on that case.

Reply:
There are many things to be revisited, also the reliance on the DSCI itself.